redirects.json

```diff
@@ -2773,55 +2755,55 @@
   },
   {
     "source":"/lib-v1/push-notifications/identify-user/q/platform/react-native/",
-    "target":"/react-native/prev/build-a-backend/identify-user/",
+    "target":"/react-native/prev/build-a-backend/push-notifications/identify-user/",
     "status":"301",
     "condition":null
   },
   {
     "source":"/lib-v1/push-notifications/interact-with-notifications/q/platform/react-native/",
-    "target":"/react-native/prev/build-a-backend/interact-with-notificiations/",
+    "target":"/react-native/prev/build-a-backend/push-notifications/interact-with-notifications/",
     "status":"301",
     "condition":null
   },
   {
     "source":"/lib-v1/push-notifications/receive-device-token/q/platform/react-native/",
-    "target":"/react-native/prev/build-a-backend/receive-device-token/",
+    "target":"/react-native/prev/build-a-backend/push-notifications/receive-device-token/",
     "status":"301",
     "condition":null
   },
   {
     "source":"/lib-v1/push-notifications/request-permissions/q/platform/react-native/",
-    "target":"/react-native/prev/build-a-backend/request-permissions/",
+    "target":"/react-native/prev/build-a-backend/push-notifications/request-permissions/",
     "status":"301",
     "condition":null
   },
   {
     "source":"/lib-v1/push-notifications/setup-push-service/q/platform/react-native/",
-    "target":"/react-native/prev/build-a-backend/set-up-push-services/",
+    "target":"/react-native/prev/build-a-backend/push-notifications/set-up-push-service/",
     "status":"301",
     "condition":null
   },
   {
     "source":"/lib-v1/push-notifications/testing/q/platform/react-native/",
-    "target":"/react-native/prev/build-a-backend/push-ntifications/test-notifications/",
+    "target":"/react-native/prev/build-a-backend/push-notifications/test-notifications/",
     "status":"301",
     "condition":null
   },
   {
     "source":"/lib-v1/q/platform/android/",
-    "target":"/android/prev/tools/libraries/",
+    "target":"/android/prev/build-a-backend/",
     "status":"301",
     "condition":null
   },
   {
     "source":"/lib-v1/q/platform/flutter/",
-    "target":"/flutter/prev/tools/libraries/",
+    "target":"/flutter/prev/build-a-backend/",
     "status":"301",
```

Comment:
there's no content in `/prev/tools/libraries/` so it just shows a blank page. redirected to the `build-a-backend` overview page.

Reply:
We'll fix this through content udpdate